frontend/views/containers/sidebar/GroupMembers.vue

```diff
@@ -45,19 +46,26 @@
           )
             | We are waiting for {username} to join the group by using their unique invite link.
 
-      menu-parent(
-        v-else
-      )
+      menu-parent(v-else)
```

Comment:
Boyscoutting: Above this, online 47, there doesn't need to be a pipe `|` i don't think. Having that there is inconsistent with how `i18n` is normally used and will complicate the parser.

Remove the pipe if it isn't needed, or if it is needed, because of some dumb pug rules, then place the string after the ) on line 46

Reply:
It's needed for the way of pug works. But I moved to the line before.